spec/support/capybara.rb

```diff
@@ -5,19 +5,23 @@
   nil
 end
 
-Capybara.register_driver :headless_chrome do |app|
+view_mode = ENV.fetch("VIEW", false) == "true" ? :chrome : :headless_chrome
+args = %w[disable-dev-shm-usage no-sandbox window-size=1280x1280]
+args = args << "headless" unless view_mode == :chrome
```

Comment:
I've added this so it's easier to view feature specs live in a browser, just prefix the ENV variable VIEW when running a spec, e.g.: 

```terminal
VIEW=true be rspec spec/features/ecp_eligible_now_reminder_set_spec.rb
```
_be = bundle exec_

Reply:
to disable, just set `VIEW=false` (or anything but `true`)